packages/make/utils/baseHandler.ts

```diff
@@ -0,0 +1,59 @@
+import * as fs from "fs";
+import * as dir from "node-dir";
+/**
+ * @returns storeBase() , getDependenencyTree(), getChangedTree(), builder()
+ *
+ */
+
+const log = (text) => { process.stdout.write(text); };
```

Comment:
actually I don't think you need brackets here

Reply:
Having brackets is my habit, Removed now 👍